cli/scorecard/cmd.go

```diff
@@ -69,10 +71,10 @@ var Cmd = &cobra.Command{
 	`,
 	Args: cobra.NoArgs,
 	PreRunE: func(cmd *cobra.Command, args []string) error {
-		if (flags.bucketName == "" && flags.dirPath == "" && !flags.stdin) ||
-			(flags.bucketName != "" && flags.stdin) ||
-			(flags.bucketName != "" && flags.dirPath != "") ||
-			(flags.dirPath != "" && flags.stdin) {
+		if (flags.bucketName == "" && flags.dirPath == "" && !flags.stdin && flags.concurrency) ||
```

Comment:
concurrency should not be included here, since it has no impact on the input source.